docs/02-app/02-api-reference/01-components/image.mdx

```diff
@@ -424,6 +425,14 @@ module.exports = {
 }
 ```
 
+### `overrideSrc`
+
+Search engines often use the `src` attribute as the canonical url for an image.
+
+When upgrading an existing website from `<img>` to `<Image>`, you may wish to maintain the same `src` attribute for SEO purposes such as image ranking or avoiding recrawl.
+
+You can use the `overrideSrc` property to set the `src` attribute instead of using the fallback that was automatically generated from the `src` prop input.
```

Comment:
Maybe include a code snippet? It's not clear to me here either which one is the image that gets displayed, versus the image that is for SEO reasons.

Reply:
@leerob I pushed an update, please take a look